suites/quincy/rbd/tier-3_rbd_snap_clone_operations.yaml

```diff
@@ -0,0 +1,104 @@
+# Tier3: Test suite to cover RBD clone related tests in scale
+# Tier-level: 3
+# Test-Suite: tier-3_rbd_snap_clone_operations.yaml
+#
+# Ceph RBD component.
+#
+# Cluster Configuration:
+#    Conf file - Baremetal config
+#    Node 10 must to be a client node
+tests:
+
+  # Setup the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node10
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Configure client node
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+      polarion-id: CEPH-83573758
+
+  - test:
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      name: Install rbd-nbd
+      config:
+        sudo: true
+        commands:
+          - "rm -rf /etc/yum.repos.d/epel*"
+          - "dnf install rbd-nbd -y"
+
+  - test:
```

Comment:
Does it need separate tets suite?
Does it create one pool, one image on one snap that many clones?

Reply:
yes, previous testsuite was about 

per pool, per image -> snap at scale eg 500

This test suite is about

per pool, per image, per snap -> clone at scale eg 500.

Since "snap protect + clone + flatten + snap unprotect" is too tightly coupled, had to do few workarounds to fit in.